DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/CTF.h

```diff
@@ -43,12 +43,12 @@ struct CompressedDigits {
   std::vector<uint16_t> bcInc;    // increment in BC if the same orbit, otherwise abs bc
   std::vector<uint32_t> orbitInc; // increment in orbit
   std::vector<uint8_t> nChan;     // number of fired channels
-  // std::vector<uint8_t> eventFlags;     // special flags about event conditions: pile-up, not use for collision time, not use for event plane, etc.
+  std::vector<uint8_t> eventFlags; // special flags about event conditions: pile-up, not use for collision time, not use for event plane, etc.
 
   // channel data
   std::vector<uint8_t> idChan;   // channels ID: 1st on absolute, then increment
   std::vector<int16_t> cfdTime;  // CFD time
-  std::vector<int16_t> qtcAmpl;  // Amplitude
+  std::vector<int32_t> qtcAmpl;  // Amplitude
```

Comment:
You wrote me that the amplitude is limited to +-2048, what is the reason to change it to int?

Reply:
Its sum amplitude from A and C sides in TCM, 18 bits long